pontoon/contributors/views.py

```diff
@@ -193,11 +220,26 @@ def settings(request):
     else:
         custom_homepage_locale = default_homepage_locale
 
+    default_preferred_source_locale = Locale(name='Default project locale', code='')
+    preferred_locales = list(Locale.objects.all())
+    preferred_locales.insert(0, default_preferred_source_locale)
+
+    # Set preferred source locale
+    preferred_source_locale = request.user.profile.preferred_source_locale
+    if preferred_source_locale:
+        custom_preferred_source_locale = (
```

Comment:
Nit: I would just call this `preferred_source_locale`.

And above, I'd rename `custom_homepage` to `custom_homepage_locale`.

Reply:
Whoops, false alarm. Sorry!